### PR TITLE
Updated Pipeline component for serverless

### PR DESCRIPTION
**Pull request by vikasagrawal-ms**
Import model  Pipeline component for Foundation Model is updated to handle serverless requests

### CODE REVIEW

File: training/model_management/components/import_model/spec.yaml [hunk 1]
```diff
@@ -104,8 +119,10 @@ outputs:
 
 jobs:
   download_model:
-    component: azureml:download_model:0.0.3
+    component: azureml://registries/azureml/components/download_model/versions/0.0.3
```
Comment (elesel) [inline training/model_management/components/import_model/spec.yaml:122]: Why expand these? The current approach (`azureml:download_model:0.0.3`) attempts to resolve the component in the registry where the pipeline component is being created.

File: training/model_management/components/import_model/spec.yaml [hunk 2]
```diff
@@ -4,18 +4,33 @@ type: pipeline
 name: import_model
 display_name: Import model
 description: Import a model into a workspace or a registry
-version: 0.0.6
+version: 0.0.5
```
[the remark this answers is not in the document]
Comment (vikasagrawal-ms) [reply training/model_management/components/import_model/spec.yaml:7]: registry has 0.0.4 as the latest so, fixing it to 0.0.5

[the remark this answers is not in the document]
Comment (elesel) [reply training/model_management/components/import_model/spec.yaml:7]: Cool. I'll let your team take it from here.

File: training/model_management/components/import_model/spec.yaml [hunk 3]
```diff
@@ -4,18 +4,33 @@ type: pipeline
...
 
 # Pipeline inputs
 inputs:
   # pipeline specific compute
   compute:
     type: string
-    description: Common compute for model download, mlflow conversion and registration
+    optional: true
+    default: serverless
```
Comment (skasturi) [inline training/model_management/components/import_model/spec.yaml:15]: > default: serverless [](http://example.com/codeflow?start=3&length=20)

Could we please check with Surya on the transition timelines?

**Review by elesel — APPROVED** [review]

**Review by HrishikeshGeedMS — APPROVED** [review]

File: training/model_management/components/import_model/spec.yaml (hunk 3 above)
Comment (vikasagrawal-ms) [reply training/model_management/components/import_model/spec.yaml:15]: Confirmed from Surya , since it is in PuP there is no whitelisting needed

**Pull request merged**